node/service/src/lib.rs

```diff
@@ -522,11 +526,15 @@ where
 
 	let frontier_backend = open_frontier_backend(client.clone(), config, rpc_config)?;
 	let frontier_block_import = FrontierBlockImport::new(client.clone(), client.clone());
+	let parachain_block_import = cumulus_client_consensus_common::ParachainBlockImport::new(
+		frontier_block_import,
+		backend.clone(),
+	);
```

Comment:
This should be a conditional. `if dev_service { ... }`. You do not want the `FrontierBlockImport` in the dev service.